### ISSUE TITLE
Radio input groups need association with corresponding question

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2195]
See Notification settings overlay, but check elsewhere too. Markup should be updated to use the [`fieldset`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) element, with the question wrapped in a `legend` tag.
